components/application-connectivity-validator/cmd/applicationconnectivityvalidator/options.go

```diff
@@ -6,42 +6,45 @@ import (
 )
 
 type options struct {
-	proxyPort              int
-	tenant                 string
-	group                  string
-	eventServicePathPrefix string
-	eventServiceHost       string
-	appRegistryPathPrefix  string
-	appRegistryHost        string
+	proxyPort                int
+	tenant                   string
+	group                    string
+	eventServicePathPrefixV1 string
+	eventServicePathPrefixV2 string
+	eventServiceHost         string
+	appRegistryPathPrefix    string
+	appRegistryHost          string
 }
 
 func parseArgs() *options {
 	proxyPort := flag.Int("proxyPort", 8080, "Proxy port.")
 	tenant := flag.String("tenant", "", "Name of the application tenant")
 	group := flag.String("group", "", "Name of the application group")
-	eventServicePathPrefix := flag.String("eventServicePathPrefix", "/v1/events", "Prefix of paths that will be directed to the Event Service")
+	eventServicePathPrefixV1 := flag.String("eventServicePathPrefixV1", "/v1/events", "Prefix of paths that will be directed to the Event Service V1")
+	eventServicePathPrefixV2 := flag.String("eventServicePathPrefixV2", "/v2/events", "Prefix of paths that will be directed to the Event Service V2")
```

Comment:
Please add the description of the new flag in the README file.

Reply:
@Szymongib Thanks for the review, I'll, the problem is that my local `goimports` doesn't show any errors!

Reply:
```
√ go test
√ goimports and fmt
√ go vet ./cmd/... 
√ go vet ./internal/... 
```

Reply:
That is probably due to a different version than the one used in buildpack.